src/components/inputs/localized-text-input/localized-text-input.js

```diff
@@ -133,25 +136,17 @@ const LocalizedTextInput = props => {
     Object.keys(props.value)
   );
 
-  const toggleLanguages = React.useCallback(() => {
-    setAreLanguagesExpanded(!areLanguagesExpanded);
-  }, [areLanguagesExpanded]);
-
   const id = getFieldId(props, {}, sequentialId);
 
   const hasErrorInRemainingLanguages =
     props.hasError ||
     getHasErrorOnRemainingLanguages(props.errors, props.selectedLanguage);
 
   if (hasErrorInRemainingLanguages) {
-    const shouldLanguagesExpand =
```

Comment:
this was unnecessary. we are already inside of an if for `hasErrorInRemainingLanguages`, so there's no point of this